### PR TITLE
Add downloads page with presentations

### PR DESCRIPTION
**Pull request by ngfreiter**
This adds a downloads page with several of the connectathon powerpoints.

### CODE REVIEW

File: content/downloads/_index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: Downloads
+menu:
+  main:
+    weight: 50
+---
+
+<!-- {{% blocks/lead color="primary"%}}
+# Resources
+Check out the resources below for more information on using SUSHI.
+{{% /blocks/lead %}} -->
```
Comment (cmoesel) [inline content/downloads/_index.md:11]: What's this commented out code for?

**Review by cmoesel — COMMENTED** [review]
Looks good, works well, but did you mean to leave that commented out block in there?

**Comment by cmoesel** [comment]
I passed the test!

File: content/downloads/_index.md (hunk 1 above)
Comment (ngfreiter) [reply content/downloads/_index.md]: I goofed, the page was more involved at first, but I decided to just keep it simple and commented that out. Removed in 6424f06.

**Review by cmoesel — APPROVED** [review]

**Pull request merged**